content/blog/2021/11/deep-dive-into-policy-logging.md

```diff
@@ -0,0 +1,218 @@
+---
+title: Deep Dive into policy logging
+authors:
+    - Rafael Fernández López
+date: 2021-11-15
+---
+
+Policies are regular programs. As such they have the need to log often times. In general, we are
+used to make our programs log into standard output (stdout) and standard error (stderr) outputs.
+
+However, policies run in a confined WebAssembly environment. For this mechanism to work as usual
+Kubewarden would need to set up the runtime environment in a way that stdout and stderr file
+descriptors are set up so that the policy can write to them, and upon completion, Kubewarden can
```

Comment:
Nit:

I would reword it to:
 "Kubewarden would need to set up the runtime environment in a way that the policy can write to stdout and stderr file
descriptors, and upon completion, Kubewarden can.."

(the 2 chained "set up" and the long subordinate sentence seems difficult to parse for me)